app/views/students/sped_referral.pdf.erb

```diff
@@ -1,61 +1,192 @@
+
+* {
+  font-family: 'Open Sans', sans-serif !important;
+}
+
+.data-table  {border-collapse:collapse;border-spacing:0;border-color:#aaa;}
+.data-table tr:nth-child(even) {background-color:white;vertical-align:top}
+.data-table tr:nth-child(odd) {background-color:lightgray;vertical-align:top}
+.data-table td{font-size:14px;padding:10px 5px;border-style:solid;border-width:0px;word-break:normal;border-color:#aaa;color:#333;border-top-width:1px;border-bottom-width:1px;vertical-align:top;}
+.data-table th{font-size:14px;font-weight:bold;padding:10px 5px;border-style:solid;border-width:0px;word-break:normal;border-color:#aaa;color:#fff;background-color:#333;border-top-width:1px;border-bottom-width:1px;}
+
+ul, li {
+  font-size: 14px;
+  margin: 0 0 0 0;
+  -webkit-padding-start: 0;
+  padding: 0px;
+  list-style-type: none;
+}
+
+</style>
+
+
+<div style="width: 768px;">
+ 
+
+<!-- Demographic Information -->
+<div style="float: left; width: 512px;">
+  <div>
+    <span style="font-size:24px; font-weight:bold;"><%= "#{@student.first_name} #{@student.last_name}"%></span>
+  </div>
+  <div style="float: left; width: 200px;">
+    <ul>
+      <li>DOB: <%= format_date(@student.date_of_birth) %></li>
+      <li><%= "Grade #{@student.grade}" %></li>
+      <li><%= @student.try(:school).try(:name) %></li>
+      <li><%= @student.try(:homeroom).try(:name) %></li>
+      <li><%= @student.program_assigned %></li>
+    </ul>
+  </div>
+  <div style="float: left; width: 312px;">
+    <ul>
+      <li>Low Income: <%= @student.free_reduced_lunch %></li>
+      <li>Language: <%= @student.home_language %></li>
+      <li>Disability: <%= @student.disability || "None"%></li>
+      <li>504 plan: <%= @student.plan_504 %></li>
+      <li>English Proficiency: <%= @student.limited_english_proficiency %></li>
+    </ul>
+  </div>
+</div>
+<!-- future photo div -->
+<div style="float: left; width: 256px;"></div>
+<br style="clear: left;" /><br/><br/>
+
+
+<!-- Notes -->
+<div>
+<% right = false %>
+<h4>Notes:</h4>
+  <% if @event_notes.any? %>
+    <% @event_notes.each do |event_note| %>
+      <div style="float: left; width:370px; padding:5px">
+        <ul>
+          <li><span style="font-weight:bold;"><%= format_date(event_note.recorded_at) %></span></li>
+          <li><span style="font-style:italic;"><%= event_note.try(:event_note_type).try(:name) %></span></li>
+          <li><%= event_note.text %></li>
+        </ul>
+      </div>
+      <% if right %>
+        <br style="clear: left;" /><br/>
+      <% end %>
+        <% right = !right %>
+    <% end %>
+  <% else %>
+    <span style="font-style:italic;">No Notes</span>
+  <% end %>
+</div>
+<br style="clear: left;" /><br/>
+
+<!-- Services -->
+<% right = false %>
+<div>
+<h4>Services:</h4>
+  <% if @services.any? %>
     <% @services.each do |service| %>
-      <li><%= "Since #{format_date(service.date_started)} - #{service.service_type.name}" %></li>
+      <div style="float: left; width:370px; padding:5px">
```

Comment:
I was wondering if it these were `display: inline-block`, would the layout wrap on its own, without having to set and clear floats manually?  Or maybe there's something whack with the way CSS works in the PDF gem.